CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contribute to ResumeWorded
+
+Welcome to ResumeWorded ! We're excited that you're interested in contributing. Please take a moment to read this guide to understand how you can contribute to the project !
+
+## Code of Conduct 
+
+Before you start contributing, We expect all contributors to follow these guidelines to create a respectful and inclusive environment for everyone.
+
+## How to Contribute ?
+
+### Submitting an issue :-
+
+If you feel ,there's any bug or feature improvement Please raise an issue.
+We are here to improve 🙌
+
+### Submitting Pull Request :-
+If you'd like to contribute code or documentation changes, you can submit a pull request (PR) to our repository. Here's how:
+
+1. Fork the repository to your GitHub account.
```

Comment:
It would be great if we have picture showing these steps.
many people are new to oss so that would be helpful

Reply:
Okay sure ! I will add these , If I have any doubt I will let you know .